Gopkg.toml

```diff
@@ -65,3 +65,7 @@
   go-tests = true
   unused-packages = true
   non-go = true
+
+[[override]]
```

Comment:
Why is "override" needed?

Reply:
I'm not sure why, but when I was running `dep ensure -update` it would break. After looking around I found this [issue](https://github.com/kubernetes-sigs/kubebuilder/issues/316) so I applied the workaround I found there.